revisionize.php

```diff
@@ -122,21 +123,39 @@ function create_revision($post, $is_original=false) {
     delete_post_meta($post->ID, '_post_original');
   }
 
+	delete_post_meta($new_id,'custom_permalink');  // NEW
+
+
+
+  wp_update_post( array(
```

Comment:
what is this for?

Reply:
The preview link wasn't working before this (in the revision). So I added this. But I'm not 100% sure about this.